components/jpa/common/src/main/java/com/kumuluz/ee/jpa/common/injection/JpaService.java

```diff
@@ -40,6 +43,8 @@
 @Priority(1)
 public class JpaService implements JpaInjectionServices {
 
+    private Logger log = Logger.getLogger(JpaService.class.getSimpleName());;
+
```

Comment:
Remove double semicolon. use capital name for the logger - "LOG".

Reply:
You can make it static and final.